README.md

```diff
@@ -1,54 +1,69 @@
+<picture align="center">
+  <img src="https://user-images.githubusercontent.com/45081667/227194559-57c73a9e-51df-4d42-b9f0-8b6dd011ae7b.png" alt="Header" >
+</picture>
+
 <p align="center">
   <a href="https://1password.com">
-    <img alt="1Password logo" src="https://user-images.githubusercontent.com/7430639/205342015-46801fd8-6701-482f-9da9-e21e7e39b3a1.svg" height="64">
-    <h3 align="center">1Password Shell Plugins</h3>
+    <h1 align="center">1Password Shell Plugins</h1>
```

Comment:
The `h1` and `h2` tags add an additional line, which looks a bit messy in combination with the line below:

<img width="995" alt="image" src="https://user-images.githubusercontent.com/7430639/227497740-979104a4-0f95-48d4-bfd7-927ba093d6b2.png">



I think the current one looks a bit cleaner:

<img width="919" alt="image" src="https://user-images.githubusercontent.com/7430639/227497324-c80f897a-1542-469a-96fa-fe454c13160e.png">

Maybe try leaving it an `h3` tag instead?

Reply:
Changed it back to h3